.gitignore

```diff
@@ -3,4 +3,5 @@ out/
 target/
 node_modules/
 Cargo.lock
-
+.vscode/
+script/
```

Comment:
Did you mean to add script/ to .gitignore?  We already have tracked files there that seem useful to have

Reply:
Good catch, reverting